### ISSUE TITLE
Navigation cube cannot be moved on 0.21.2

### ISSUE THREAD

**Issue by fenrir-5** [FreeCAD/FreeCAD#13969]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Hi everyone,
After a quick search, I found few relevant topics on FreeCAD forum but the problem seem to be related to AMD graphics. I'm using Intel graphics and everything worked great few months ago when I last used FreeCAD.

### Full version info

```shell
[code]
OS: Arch Linux (XFCE/xfce)
Word size of FreeCAD: 64-bit
Version: 0.21.2.33771 (Git)
Build type: Release
Branch: makepkg
Hash: b9bfa5c5507506e4515816414cd27f4851d00489
Python 3.12.3, Qt 6.7.0, Coin 4.0.2, Vtk 9.3.0, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * LCInterlocking
  * A2plus 0.4.62
  * sheetmetal 0.4.0
  * ExplodedAssembly
  * Render 2024.1.23
  * ArchTextures
[/code]
```

### Subproject(s) affected?

Other (specify in description)

### Anything else?

![anim](https://github.com/FreeCAD/FreeCAD/assets/18752619/0ca52008-ee3f-4b5d-a8d4-643409f457d9)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
Is the setting movable navigation cube checked?
Could you try if the issue persists in the current weekly builds: https://github.com/FreeCAD/FreeCAD-Bundle/releases/tag/weekly-builds

**Comment by fenrir-5** [comment]
That option seem to be missing in Preferences > Display > Navigation tab > Navigation cube section. In report view there are some error which I don't think they have something to do with the Navigation cube situation:
`
12:52:14  Path override failed for key base::DIR_APP_DICTIONARIES and path '/usr/lib/freecad/bin/qtwebengine_dictionaries'
12:52:14  The `.FCStd#` backup format is deprecated as of v0.21 and may be removed in future versions.
To update, check the 'Preferences->General->Document->Use date and FCBak extension' option.
`

I'm a bit scared of trying weekly builds, or at least not until I finish the project I'm working on now.

**Comment by liamhays** [comment]
This is probably the same issue as this: https://gitlab.archlinux.org/archlinux/packaging/packages/freecad/-/issues/3, which is a product of Arch's switching the FreeCAD build to Qt6.

**Comment by maxwxyz** [comment]
could this be the same issue as https://github.com/FreeCAD/FreeCAD/issues/13494#issuecomment-2106262038

**Comment by fenrir-5** [comment]
Yep, i'm sure that's the issue. Problem is I can't downgrading it without falling into the Python rabbit hole so I guess waiting for the next release is best I can do.

With the 0.21.2-6 I get:
`freecad: error while loading shared libraries: libpython3.11.so.1.0: cannot open shared object file: No such file or directory`

**Comment by maxwxyz** [comment]
I'll close this then, you can comment in the other issue

**Comment by maxwxyz** [comment]
Duplicate of #13494
